frontend/src/components/Organizations.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import Loader from "./Loader";
+import { User } from "../services/auth";
+import * as api from "../api";
+
+type OrgsProps = {
+  user: User;
+};
+
+type OrgsState = {
+  done: boolean;
+  orgs: string[];
+};
+
+class Organizations extends React.Component<OrgsProps, OrgsState> {
+  state: OrgsState = {
+    done: false,
+    orgs: []
+  };
+
+  componentDidMount() {
+    return api
+      .orgs()
+      .then(resp => {
+        this.setState({ 
+          done: true,
+          orgs: resp,
+         });
+      })
+      .catch(err => {
+        console.error(err);
+      });
```

Comment:
let's notify user about error. Now it continues showing loader.

Reply:
:+1: I didn't want to deal with UI for errors for now, but it's so simple it makes sense to include. Done in 4cbfb29.